frontend/src/components/Forms/InterestsForm/index.js

```diff
@@ -1,65 +1,58 @@
 import { PropTypes } from "prop-types";
 import { OTHER, SELECTED_INTERESTS } from "pages/SignupPage";
 
-import Checkbox from "components/Checkbox";
+import CheckboxWithIcon from "components/CheckboxWithIcon";
 import FormLayout from "components/Forms/FormLayout";
-import Input from "components/Input";
+
+import development from "assets/svgs/checkboxes/development.svg";
+import design from "assets/svgs/checkboxes/design.svg";
+import pm from "assets/svgs/checkboxes/pm.svg";
+import other from "assets/svgs/checkboxes/other.svg";
 
 import styles from "./index.module.css";
 
-const InterestsForm = ({
-  handleChange,
-  validatesChecked,
-  inputHidden,
-  extraInterest,
-}) => {
+const InterestsForm = ({ handleChange, validatesChecked }) => {
   return (
     <FormLayout
       headerText="What are you interested in?"
       subtitleText="In case you want to know job opportunities"
     >
       <>
         <div className={styles.selectedInterestsWrapper}>
-          <Checkbox
+          <CheckboxWithIcon
             id="development"
+            icon={development}
             label="Development"
             checked={validatesChecked("development")}
             onChange={handleChange(SELECTED_INTERESTS)}
           />
-          <Checkbox
+          <CheckboxWithIcon
             id="design"
+            icon={design}
             label="Design"
             checked={validatesChecked("design")}
             onChange={handleChange(SELECTED_INTERESTS)}
           />
-          <Checkbox
+          <CheckboxWithIcon
             id="product-management"
-            label="Product Management"
+            icon={pm}
+            label="PM"
             checked={validatesChecked("product-management")}
             onChange={handleChange(SELECTED_INTERESTS)}
           />
-          <Checkbox
+          <CheckboxWithIcon
             id={OTHER}
+            icon={other}
             label="Other"
             checked={validatesChecked(OTHER)}
             onChange={handleChange(OTHER)}
           />
```

Comment:
You can create an object for this data and loop over it. Kinda similar to what you're doing with `stepInfo` in the `SignupPage`